src/Migrations/Version20200521165709.php

```diff
@@ -0,0 +1,42 @@
+<?php declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20200521165709 extends AbstractMigration
+{
+    public function up(Schema $schema) : void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'postgresql', 'Migration can only be executed safely on \'postgresql\'.');
+
+        $this->addSql("UPDATE dataset_submission SET temporal_extent_begin_position = temporal_extent_begin_position AT TIME ZONE 'UTC'");
+
+        $this->addSql("UPDATE dataset_submission SET temporal_extent_end_position = temporal_extent_end_position AT TIME ZONE 'UTC'");
+
+        $this->addSql('ALTER TABLE dataset_submission ALTER temporal_extent_begin_position TYPE DATE');
+        $this->addSql('ALTER TABLE dataset_submission ALTER temporal_extent_begin_position DROP DEFAULT');
+        $this->addSql('ALTER TABLE dataset_submission ALTER temporal_extent_end_position TYPE DATE');
+        $this->addSql('ALTER TABLE dataset_submission ALTER temporal_extent_end_position DROP DEFAULT');
+    }
+
+    public function down(Schema $schema) : void
+    {
+        // this down() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'postgresql', 'Migration can only be executed safely on \'postgresql\'.');
+
+        $this->addSql('ALTER TABLE dataset_submission ALTER temporal_extent_begin_position TYPE TIMESTAMP(0) WITH TIME ZONE');
+        $this->addSql('ALTER TABLE dataset_submission ALTER temporal_extent_begin_position DROP DEFAULT');
+        $this->addSql('ALTER TABLE dataset_submission ALTER temporal_extent_end_position TYPE TIMESTAMP(0) WITH TIME ZONE');
+        $this->addSql('ALTER TABLE dataset_submission ALTER temporal_extent_end_position DROP DEFAULT');
+
+        $this->addSql("UPDATE dataset_submission SET temporal_extent_begin_position = temporal_extent_begin_position AT TIME ZONE '+10'");
```

Comment:
I thought you didn't want to just target the middle of the day ;-) j/k. cool.

Reply:
10 hours is 5 hours from migrating, plus 5 hours to bring it to UTC. The timezone is not stored as a name but in ISO format.